browse/templates/abs/labs_tabs.html

```diff
@@ -160,6 +160,21 @@ <h1>Recommenders and Search Tools</h1>
             <span id="label-for-core">CORE Recommender</span> <em>(<a href="https://labs.arxiv.org/">What is CORE?</a>)</em>
           </div>
         </div>
+        {%- if abs_meta.primary_category in ['astro-ph', 'hep-ph', 'hep-th', 'gr-qc'] %}
+        <div class="columns is-mobile lab-row">
+          <div class="column lab-switch">
+            <label class="switch">
+              <input id="iarxiv-recommender-toggle" type="checkbox" class="lab-toggle" aria-labelledby="label-for-iarxiv">
+              <span class="slider"></span>
+              <span class="is-sr-only">IArxiv recommender toggle</span>
+            </label>
+          </div>
+          <div class="column lab-name">
+            <span id="label-for-iarxiv">IArxiv Recommender</span> <em>(<a href="/iarxiv_login">Email Signup</a> | <a href="https://iarxiv.org/about">What is IArxiv?</a>))</em>
+          </div>
+        </div>
+        {% endif -%}
+
       </div>
       <div style="min-height: 15px" id="connectedpapers-output"></div>
       <div id="coreRecommenderOutput"></div>
```

Comment:
I think were missing the div for the IArxiv output... and my understanding is that the output will be a link to the IArxiv login page on arxiv.org, plus a longer description. Since that output is so simple we might just build it as html right on this page instead of using a script to generate it like the other labs projects do. 

Do you think that sounds right @bmaltzan ?

Reply:
Hey @SBBCornell , yes, I didn't create a div for any output to be displayed, when the toggle is on. Sure, just a link to arxiv.org/iarxiv_login is need, and simple html that is toggled visible sounds ideal.

Reply:
I added the toggle. It's pretty simple, but I did end up adding it in a js script. It might be easier to read that way, just following the existing conventions. What do you think?

Reply:
looks good!